src/utils/vcs.rs

```diff
@@ -249,15 +249,41 @@ fn find_matching_rev(
         if let Some(url) = remote.url();
         then {
             if !discovery || is_matching_url(url, &reference_url) {
-                debug!("  found match: {} == {}", url, &reference_url);
+                debug!("  found match: {} == {}, {:?}", url, &reference_url, r);
                 let head = repo.revparse_single(r)?;
+                if let Some(tag) = head.as_tag(){
+                    debug!("  tag.target(): {:?}", tag.target());
+                    if let Ok(tag_commit) = tag.target() {
+                        return Ok(Some(log_match!(tag_commit.id().to_string())));
+                    }
+                }
+                debug!("  not a tag");
```

Comment:
This logging is a bit verbose, even for `debug`. Mind if we remove these two additional logs again or make them `trace!`?

Reply:
Removed.